File: examples/plugin-archives/demo-table-tools/python/manifest-table.py
def execute(params, upstream, context):
    count = max(1, int(params.get('count', 6)))
    start = float(params.get('start', 1))
    step = float(params.get('step', 2))
    return pd.DataFrame({'index': list(range(count)), 'value': [start + i * step for i in range(count)]})

File: examples/plugin-archives/demo-table-tools/js/manifest-table.js
function execute(params, upstream, context, api) {
  const count = Math.max(1, Math.trunc(Number(params.count ?? 6)));
  const start = Number(params.start ?? 1);
  const step = Number(params.step ?? 2);
  return api.Table.fromRecords(Array.from({ length: count }, (_, index) => ({ index, value: start + index * step })));
}
